packages/repository/src/decorators/model.ts

```diff
@@ -81,14 +81,28 @@ export function property(definition?: Partial<PropertyDefinition>) {
 export namespace property {
   export const ERR_PROP_NOT_ARRAY =
     '@property.array can only decorate array properties!';
+  export const ERR_NO_ARGS = 'Should have at least one argument';
```

Comment:
This error message is too generic which may make it difficult for users to understand what went wrong.

Proposed text:

```
@property.oneOf requires at least one argument.
```

`oneOf` with a single item does not make much sense to me - people should use `Foo` instead of `oneOf: [Foo]`. I would personally enforce at least two arguments.

Reply:
Actually, the JSON Schema specification says that one item is fine:
>An instance validates successfully against this keyword if it validates successfully against exactly one schema defined by this keyword's value.

[Source](http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.7.3)